docs/nodes/build/launch-an-avalanche-validator-on-aws-with-one-click.md

```diff
@@ -0,0 +1,345 @@
+and the [AWS Privacy Notice](https://aws.amazon.com/privacy/). After reviewing
+these you want to click on the "Continue to Configuration" button.
+
+## Configure This Software
+
+This page lets you choose a fulfillment option and software version to launch
+this software. No changes are needed as the default settings are sufficient.
+Leave the `Fulfillment Option` as `64-bit (x86) Amazon Machine Image (AMI)`. The
+software version is the latest build of [the AvalancheGo full
+node](https://github.com/ava-labs/avalanchego/releases), `v1.9.5 (Dec 22, 2022)`, AKA `Banff.5`. 
+Also, the Region to deploy in can be left as `US East (N.  Virginia)`. On the right you'll 
+see the software and infrastructure pricing. Lastly, click the "Continue to Launch" button.
+
+## Launch This Software
+
+Here you can review the launch configuration details and follow the instructions
+to launch the Avalanche Validator Node. The changes are very minor. Leave the
+  action as "Launch from Website." The EC2 Instance Type should remain
+`c5.2xlarge`. The primary change you'll need to make is to choose a keypair
+which will enable you to `ssh` into the newly created EC2 instance to run `curl`
+commands on the Validator node. You can search for existing Keypairs or you can
+create a new keypair and download it to your local machine. If you create a new
+keypair you'll need to move the keypair to the appropriate location, change the
+permissions and add it to the OpenSSH authentication agent. For example, on
+MacOS it would look similar to the following: 
+
+```zsh
+# In this example we have a keypair called avalanche.pem which was downloaded from AWS to ~/Downloads/avalanche.pem
+# Confirm the file exists with the following command
+test -f ~/Downloads/avalanche.pem && echo "avalance.pem exists."
+
+# Running the above command will output the following:
+# avalance.pem exists.
+
+# Move the avalanche.pem keypair from the ~/Downloads directory to the hidden ~/.ssh directory
+mv ~/Downloads/avalanche.pem ~/.ssh
+  
+# Next add the private key identity to the OpenSSH authentication agent
+ssh-add ~/.ssh/avalanche.pem; 
+
+# Change file modes or Access Control Lists
+sudo chmod 600 ~/.ssh/avalanche.pem
+```
+
+Once these steps are complete you are ready to launch the Validator node on EC2.
+To make that happen click the "Launch" button 
+
+![launch successful](/img/one-click-validator-node/launch-successful.png)
+
+You now have an Avalanche Validator node deployed on an AWS EC2 instance! Copy
```

Comment:
```suggestion
You now have an Avalanche node deployed on an AWS EC2 instance! Copy
```
Strictly speaking, it is not a validator yet until later step.

Reply:
Resolved